2017-07-11-tidbinternal2.md

```diff
@@ -81,31 +81,33 @@ TiDB allocates a `TableID` to each table, an `IndexID` to each index, and a `Row
 Each row of data is encoded into a Key-Value pair according to the following rule:
 
 ```
-Key: tablePrefix_rowPrefix_tableID_rowID
+Key: tablePrefix_tableID_rowSeperator_rowID
```

Comment:
Perhaps use `recordPrefixSep` to match line 106.

Reply:
@kennytm ？